src/main/java/at/ac/tuwien/ifs/sge/leeroy/agents/BattleSimulator.java

```diff
@@ -37,10 +37,12 @@ public static double getWinProbability(int attackTroops, int defendTroops) {
         final double expectedLossDef = 1.08;
         final double expectedLossAtt = 0.922;
         if ((attackTroops / expectedLossAtt) > (defendTroops / expectedLossDef)) {
-            return getWinProbability(10, (int) Math.floor(defendTroops - (attackTroops - 10) / expectedLossAtt * expectedLossDef));
+            int targetV = attackTroops == 10 ? 9 : 10;
```

Comment:
What is `targetV`?

Reply:
late hour fix for an endless loop which was caused by a 10 vs 11 scenario (and my stupid heuristic).
The intermediate variable is not really relevant. It just tells what should be the target value for the recursion call